src/ipc-handlers.ts

```diff
@@ -415,17 +415,19 @@ export async function startServer(
 	try {
 		await server.start();
 	} catch ( error ) {
+		if (
```

Comment:
Would it make sense to add a comment shortly explaining that we don't need to track memory errors anymore, but we want to track unreachable instruction ones? Also, do we need to track the latter?

Reply:
Good point, I documented it in [8c09798](https://github.com/Automattic/studio/pull/1302/commits/8c09798c84fac6ddb331cba35dd166d6a927174c).

I think that it's still good to collect unreachable instruction errors in Sentry. It's technically not a Studio issue, but it's a great source of feedback for Playground and a useful metric for us as we should aim to get this error to 0.